src/macaron/config/global_config.py

```diff
@@ -60,9 +64,23 @@ def load(
         self.debug_level = debug_level
         self.local_repos_path = local_repos_path
         self.gh_token = gh_token
-        self.policy_path = policy_path
         self.resources_path = resources_path
 
+        # Find the policies
+        policy_files = []
+        for policy_path in policy_paths:
+            if os.path.isdir(policy_path):
+                for file_name in os.listdir(policy_path):
```

Comment:
I wonder if using `glob` here would be possible?

For example:
```python
for file_path in glob.iglob(f"{policy_path}/**", recursive=False)
    ...
```

Reply:
Fixed https://github.com/oracle-samples/macaron/pull/46/commits/4e61112595339f7871796e707f604cc1ff1b5711